OracleJava/java-11/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2015 Oracle and/or its affiliates. All rights reserved.
```

Comment:
You need to update the date in your Copyright statement

Reply:
Done.